client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -34,10 +34,10 @@ <h3 class="oui-header_4 oui-tile__title" data-translate="domain_dashboard_genera
                                     <div class="oui-tile__definition">
                                         <div class="oui-tile__term d-inline-block mr-2"
                                                 data-translate="domain_dashboard_dns_management"></div>
-                                        <span data-uib-tooltip="{{:: 'domain_dashboard_tooltip_dns' | translate}}"
-                                              data-uib-tooltip-append-to-body="true">
-                                              <span class="fa fa-info-circle" aria-hidden="true"></span>
-                                          </span>
+                                        <span class="fa fa-info-circle"
+                                              data-oui-tooltip="{{:: 'domain_dashboard_tooltip_dns' | translate}}"
+                                              data-oui-tooltip-placement="top">
+                                        </span>  
```

Comment:
Extra space. Does EditorConfig is well configured on your editor?

Reply:
I'll check with him with afternoon